util/chunk/disk.go

```diff
@@ -44,14 +44,14 @@ var bufReaderPool = sync.Pool{
 	New: func() interface{} { return bufio.NewReaderSize(nil, readBufSize) },
 }
 
-var tmpDir = filepath.Join(os.TempDir(), "tidb-server-"+filepath.Base(os.Args[0]))
+var tmpDir = config.GetGlobalConfig().TempStoragePath
 
 func init() {
 	err := os.RemoveAll(tmpDir) // clean the uncleared temp file during the last run.
 	if err != nil {
 		log.Warn("Remove temporary file error", zap.String("tmpDir", tmpDir), zap.Error(err))
 	}
-	err = os.Mkdir(tmpDir, 0755)
+	err = os.MkdirAll(tmpDir, 0755)
```

Comment:
The initialization of `tmpDir` and `init` function is executed before the config initialization so that `tmpDir` here is always the default value. Could you fix this problem and add a test case about it?

Reply:
Ohhh... no! sure, I will fix it soon.

Reply:
@SunRunAway Fixed. PTAL Thanks